### PR TITLE
Add Godoc badge to readme

### PR DESCRIPTION
**Pull request by RobbieMcKinstry**
This PR closes https://github.com/pulumi/pulumi-command/issues/70

I wanted to add badges for PyPi, NPM, etc but apparently, they don't offer a badge generator tool like Go does.

### CODE REVIEW

**Comment by stack72** [comment]
You can always copy the same badges we have in Pulumi-aws :)

**Comment by RobbieMcKinstry** [comment]
> You can always copy the same badges we have in Pulumi-aws :)

Great idea!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Pulumi Command Provider (preview)
 
+[![Go Reference](https://pkg.go.dev/badge/github.com/pulumi/pulumi-command.svg)](https://pkg.go.dev/github.com/pulumi/pulumi-command)
```
Comment (t0yv0) [inline README.md:3]: Hi @RobbieMcKinstry the link looks broken: https://pkg.go.dev/github.com/pulumi/pulumi-command

Any idea?

There is this https://pkg.go.dev/github.com/brandonkal/pulumi-command/sdk/go/command

Suspicious.

Comment (t0yv0) [reply README.md:3]: Perhaps https://pkg.go.dev/github.com/pulumi/pulumi-command/sdk

**Comment by stack72** [comment]
I added this in another PR we merged @t0yv0 - it's already in the README so this can be closed

**Pull request closed without merging**